Formula/format-udf.rb

```diff
@@ -0,0 +1,15 @@
+class FormatUdf < Formula
+  desc "Bash script to format a block device to UDF"
+  homepage "https://github.com/JElchison/format-udf"
+  url "https://github.com/JElchison/format-udf/archive/1.4.0.tar.gz"
+  sha256 "41f259471fce19b8b716ff73498cd9e6864ffebc3855ffb5be5d763f5397d765"
+  bottle :unneeded
+
+  def install
+    bin.install "format-udf.sh" => "format-udf"
+  end
+
+  test do
+    system "#{bin}/format-udf.sh", "-h"
```

Comment:
This is failing: http://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/2600/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/test_format_udf/

Might instead want to use `assert_match` and `shell_output` with the second argument as the expected exit code.